lib/dynamodb-integration.js

```diff
@@ -80,27 +107,13 @@ lib.upload = function(dynamodbClient, tableName, streamName, logEvents, options,
 
     debug('send to aws');
 
-    dynamodbClient.send(new BatchWriteItemCommand(payload), function (err, data) {
-        debug('sent to aws, err: ', err, ' data: ', data)
-        if (err) {
-            debug('error during batchWriteItem', err, true)
-            retrySubmit(dynamodbClient, payload, 3, cb)
-        } else {
-            cb()
-        }
-    });
+    logsQueue.add(() => sendLogsToDynamoDb(dynamodbClient, payload, cb));
 };
 
-function retrySubmit(dynamodbClient, payload, times, cb) {
+function retrySubmit(dynamodbClient, payload, cb, times) {
```

Comment:
seems like a redundant function ...

Reply:
I think we should keep it for readability. I need this function to easily and cleanly wrap the logs writing to dyanmodb and add them to our promises queue